integration_tests/pom.xml

```diff
@@ -417,17 +417,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.3.2</version>
-                <executions>
-                    <execution>
-                        <id>attach-javadoc</id>
-                        <goals>
-                            <goal>jar</goal>
-                        </goals>
-                    </execution>
-                </executions>
                 <configuration>
-                    <doclint>none</doclint>
                     <detectOfflineLinks>false</detectOfflineLinks>
                 </configuration>
             </plugin>
```

Comment:
This whole `<plugin>` definition is unnecessary. We merely need an override of the same-named user property `detectOfflineLinks` https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html#detectOfflineLinks which should be added around [L30 in this pom](https://github.com/NVIDIA/spark-rapids/pull/8234/files/52ba01f13bd17c5a53c809160097f12c0a00d307..ce4a38836024a1a2b3a59067a14e2cc84e0fbeb1#diff-4b412ae5512cbf72928bdd8bfe6aa562a04f709fcbb7d97b04b6781e338ab987R30).

Reply:
I would like to keep it as a plugin dedicated configuration override instead of user property

Reply:
Adding `detectOfflineLinks` directly could confuse people, if they do not know if property, 
and we would have to add comment to explain where this property would be used. Also it may bring some conflict if other place using it w/ the same name.

To me, it does not bring any benefit to put it in property list (saving some code?). 
Explicitly adding to the plugin config seems more maintainable IMHO.